### ISSUE TITLE
Scheduling the GAN training

### ISSUE THREAD

**Issue by Neerajj9** [senguptaumd/Background-Matting#61]
Any specific reason for training the Generator more than the Discriminator ? The code states that the Discriminator is trained after every 5 epochs. Why is it so ?

**Comment by senguptaumd** [comment]
You could change the learning rate of the discriminator and make it 5 or 10 times slower than the generator to make the training stable (This is kinda similar to updating Disc every 5 iteration). You could also linearly decreasing the lr of both Gen and Disc to make it more converge smoothly.
